### ISSUE TITLE
Version 4.5 missing?

### ISSUE THREAD

**Issue by DragonSA** [fsharp/fsharp#872]
Microsoft has [announced](https://blogs.msdn.microsoft.com/dotnet/2018/08/14/announcing-f-4-5/) the release of F# 4.5, however this repo does not have a corresponding tag.

**Comment by baronfel** [comment]
Yes, there is a tracking PR for this integration that you can follow here: https://github.com/fsharp/fsharp/pull/870

Once that is closed, we'll tag this repository and notify downstream packagers (like the mono project)
